modules/mtl/arrow-mtl-data/.kotlintest/spec_failures

```diff
@@ -0,0 +1 @@
+arrow.mtl.WriterTTest
```

Comment:
Should this be ignored by git? I don't know too much about these files tbh

Reply:
yes, there is an entry in `.git_ignore` for it. i removed it